src/ontology/reports/sync-synonym/ncit.synonyms.confirmed.robot.tsv

```diff
@@ -11467,17 +11474,17 @@ MONDO:0006309	mucinous gastric adenocarcinoma	oio:hasExactSynonym		mucinous aden
 MONDO:0006309	mucinous gastric adenocarcinoma	oio:hasExactSynonym		mucinous gastric adenocarcinoma	mucinous gastric adenocarcinoma	Mucinous Gastric Adenocarcinoma	NCIT:C5248	Gastric Mucinous Adenocarcinoma					mucinous gastric adenocarcinoma	NCIT:C5248										
 MONDO:0006309	mucinous gastric adenocarcinoma	oio:hasExactSynonym		mucinous stomach adenocarcinoma	mucinous stomach adenocarcinoma	Mucinous Stomach Adenocarcinoma	NCIT:C5248	Gastric Mucinous Adenocarcinoma					mucinous stomach adenocarcinoma	NCIT:C5248										
 MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		myelodysplastic/myeloproliferative neoplasm	myelodysplastic/myeloproliferative neoplasm	Myelodysplastic/Myeloproliferative Neoplasm	NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm					myelodysplastic/myeloproliferative neoplasm	NCIT:C27262										
-MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MDS/MPN			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				MDS/MPN	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
-MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MPD-MDS			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				MPD-MDS	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
-MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MPD/MDS			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION				MPD/MDS	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
+MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MDS/MPN			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION	http://purl.obolibrary.org/obo/mondo#ABBREVIATION			MDS/MPN	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
+MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MPD-MDS			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION	http://purl.obolibrary.org/obo/mondo#ABBREVIATION			MPD-MDS	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
+MONDO:0006311	myelodysplastic/myeloproliferative neoplasm	oio:hasExactSynonym		MPD/MDS			NCIT:C27262	Myelodysplastic/Myeloproliferative Neoplasm	http://purl.obolibrary.org/obo/mondo#ABBREVIATION	http://purl.obolibrary.org/obo/mondo#ABBREVIATION			MPD/MDS	NCIT:C27262	http://purl.obolibrary.org/obo/mondo#ABBREVIATION									
```

Comment:
Is this correct to see `http://purl.obolibrary.org/obo/mondo#ABBREVIATION` three times?

Reply:
Yes. Explanation in [comment above](https://github.com/monarch-initiative/mondo-ingest/pull/729#discussion_r1892962924).